dora/core/common/src/main/java/alluxio/wire/MissingRequiredFieldsParsingException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.Message;
+
+/**
+ * Thrown when the parsed proto is missing any required field.
+ */
+public class MissingRequiredFieldsParsingException extends ProtoParsingException {
```

Comment:
do we need additional exception for this purpose? any existing exception class to reuse?

Reply:
Not as far as I know. We've got things like `UnexpectedAlluxioException` but I think they are too general to be useful in this case.